build.sbt

```diff
@@ -88,11 +90,11 @@ lazy val commonSettings = releaseSettings ++ Seq (
 )
 
 lazy val jvmSdkDependencies = Seq (
-  resolvers += Resolver.sonatypeRepo("releases"),
+  resolvers in ThisBuild ++= Resolver.sonatypeRepo("releases") :: Resolver.mavenLocal :: Nil,
```

Comment:
exactly, before 90% of the JVM SDK releases I publish it local and try out if Sunrise works